modules/apps/frontend-js/frontend-js-dropdown-support-web/src/main/resources/META-INF/resources/isClickOutside.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export default (target, ...elements) => {
```

Comment:
Why the destructuring?

Reply:
From an older implementation 😴 . Removing! 👍

Reply:
because We can receive more than one elements to check if the click was outside the `target` element or not